core/src/main/java/org/mule/DefaultMuleMessage.java

```diff
@@ -475,18 +468,6 @@ public void setInboundProperty(String key, Object value, DataType<?> dataType)
     }
 
     @Override
-    public void setInvocationProperty(String key, Object value)
```

Comment:
DefaultMuleMessage still have setProperty methods. Should not they be private instead of public?